vault/identity_store_oidc_provider_test.go

```diff
@@ -259,6 +259,64 @@ func TestOIDC_Path_OIDC_ProviderClient(t *testing.T) {
 	}
 }
 
+// TestOIDC_Path_OIDC_ProviderClient tests CRUD operations for clients
```

Comment:
nit: comment does not match the function

Reply:
Updated